src/App.tsx

```diff
@@ -1,26 +1,33 @@
 import React from 'react';
 import './App.css';
-import { CardProperties } from './components/cards/cardProperties';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faHeart } from '@fortawesome/free-regular-svg-icons';
-import img from './components/cards/cardProperties/Svg.svg';
+import { CardBooking } from './components/cards/card-booking';
+import img from './components/cards/card-booking/img.svg';
```

Comment:
img parace generico demais. Poderia usar outro nome, o que acha? Mesma coisa para o nome do arquivo

Reply:
Sim, minha ideia é usar outro nome quando eu for utilizar o componente. Como aí só era um teste, não pensei muito sobre o nome.